### PR TITLE
CSS01task

### PR DESCRIPTION
**Pull request by vladshavel**

### CODE REVIEW

**Comment by vladshavel** [comment]
User@lugin MINGW64 ~/Desktop/myworks/sc.it-academy.by/vshavel (m-sc1-59-19)      $ git remote add my https://github.com/pluhin/sc.it-academy.by.git               fatal: remote my already exists.
--


should i just to change name "my"?

File: vshavel/CssSecond/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	font-weight:bold;
+	font-size:medium;
+}
+
+header {
+	text-align:right !important;
+}
+
+h3 {
+	text-align:left;
```
Comment (pluhin) [inline vshavel/CssSecond/style.css]: This is default value for any H1..6

File: vshavel/CssSecond/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	font-weight:bold;
+	font-size:x-large;
+	text-decoration:underline;
+	text-indent:6mm;
+	font-family: 'Pacifico', cursive;
+}
+article {
+font-family: 'Caveat', cursive;
+text-indent:6mm;
+text-align:left;
```
Comment (pluhin) [inline vshavel/CssSecond/style.css]: This default value for article tag

File: vshavel/CssSecond/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+font-weight:normal;
```
Comment (pluhin) [inline vshavel/CssSecond/style.css]: Also default

File: vshavel/CssSecond/letter.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD>
+		<META charset="utf-8">
+		<TITLE>Пример страницы</TITLE>
+	<link rel="stylesheet" href="style.css" type="text/css">
```
Comment (pluhin) [inline vshavel/CssSecond/letter.html]: This link should be below links with font import

File: vshavel/CssSecond/style.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+font-size:large;
+}
+
+footer {
+	text-align:left;
```
Comment (pluhin) [inline vshavel/CssSecond/style.css]: it is default value

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — APPROVED** [review]
Very good, approved

**Pull request merged**